chain/client/src/client.rs

```diff
@@ -445,6 +451,9 @@ impl Client {
             prev_next_bp_hash
         };
 
+        #[cfg(feature = "sandbox")]
+        let delta_time = self.sandbox_delta_time();
```

Comment:
move it closer to 511

Reply:
Had it there because of borrow rules here, but I can move the declaration of `let (at, timestamp) = ...` close to this declaration instead